.prettierignore

```diff
@@ -23,3 +23,4 @@ app/assets/javascripts/discourse/tests/fixtures
 app/assets/javascripts/discourse/tests/helpers/assertions.js
 node_modules/
 dist/
+**/*.rb
```

Comment:
I'm curious, why this change?

Reply:
prettier was checking my ruby files in vim

Reply:
It is probably better for you to specify what to run prettier on: 

`let g:ale_fixers = {'scss': ['prettier'], 'javascript': ['eslint', 'prettier'] }`

That's what I do.